setup.py

```diff
@@ -7,9 +7,9 @@
 with open(convert_path('src/version.py')) as f:
     exec(f.read(), version)
```

Comment:
Here you may change it to use regex match to avoid the usage of `exec()` to effectively shut CodeFactor up.